monitoring/src/kieker/monitoring/probe/disl/flow/operationExecution/KiekerMonitoringAnalysis.java

```diff
@@ -0,0 +1,86 @@
+package kieker.monitoring.probe.disl.flow.operationExecution;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import kieker.common.record.controlflow.OperationExecutionRecord;
+import kieker.monitoring.core.controller.IMonitoringController;
+import kieker.monitoring.core.controller.MonitoringController;
+import kieker.monitoring.core.registry.ControlFlowRegistry;
+import kieker.monitoring.core.registry.SessionRegistry;
+import kieker.monitoring.timer.ITimeSource;
+
+/**
+ * This class contains the monitoring logic that can be used by different frameworks, especially DiSL and Javassist.
+ */
+public abstract class KiekerMonitoringAnalysis {
```

Comment:
Why is this class called ..Analysis?

Reply:
Analysis doesn't make sense, I've renamed it to `OperationExecutionDataGatherer`.